stack.yaml

```
@@ -18,3 +18,4 @@ extra-deps:
   commit: e0e71bc40fcc1fa410f98a5963b9ed8eefa1e92d
   subdirs:
     - core
+- attoparsec-aeson-2.1.0.0
```

```suggestion
- attoparsec-aeson-2.1.0.0@sha256:fa83aba43bfa58490de8f274d19b9d58b6403a207b12cac5f93922102b084c52,1154
```

We prefer hashes because that way it will always build with the exact same version, which is Stack's strength.

We'll also need to add this to several Stack configs in `.ci/`.